cmd/export_accounts.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/stellar/stellar-etl/internal/input"
+	"github.com/stellar/stellar-etl/internal/transform"
+	"github.com/stellar/stellar-etl/internal/utils"
+)
+
+var accountsCmd = &cobra.Command{
+	Use:   "export_accounts",
+	Short: "Exports the account data.",
+	Long:  `Exports historical account data within the specified range to an output file.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		startNum, endNum, limit, path, useStdOut := utils.MustBasicFlags(cmd.Flags(), cmdLogger)
```

Comment:
Naming convention: `stdout` is a phrase, so the variable should be named `useStdout`.